Header header

string[] bodies
geometry_msgs/Point[] positions
hri_msgs/NormalizedPointOfInterest2D[] points2d